### PR TITLE
taxonomy: Konjac

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -46757,6 +46757,10 @@ ciqual_food_code:en:53200
 ciqual_food_name:en:Taro, tuber, raw
 ciqual_food_name:fr:Taro, tubercule, cru
 
+# description:en:Konjac is a common name of the East and Southeast Asian plant Amorphophallus konjac (syn. A. rivieri), which has an edible corm (bulbo-tuber).
+en:Konjac
+de:Konjak
+fr:Konjac
```
Comment (teolemon) [inline taxonomies/ingredients.txt:46764]:
```suggestion
```suggestion
description:en:Konjac is a common name of the East and Southeast Asian plant Amorphophallus konjac (syn. A. rivieri), which has an edible corm (bulbo-tuber).
```
```

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -46757,6 +46757,10 @@ ciqual_food_code:en:53200
 ciqual_food_name:en:Taro, tuber, raw
 ciqual_food_name:fr:Taro, tubercule, cru
 
+# description:en:Konjac is a common name of the East and Southeast Asian plant Amorphophallus konjac (syn. A. rivieri), which has an edible corm (bulbo-tuber).
```
Comment (teolemon) [inline taxonomies/ingredients.txt:46760]:
```suggestion
```

Comment (moon-rabbitOFF) [reply taxonomies/ingredients.txt:46760]: I modified it

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**